docker/docker-compose.prod.yaml

```diff
@@ -33,7 +33,7 @@ services:
           - redis
 
   celery_worker:
-    image: cll/celery_worker
+    image: cll/app
```

Comment:
**Note:** This would actually point to the Habor image in prod env

Reply:
[" We do not recommend using Watchtower in a commercial or production environment. "](https://github.com/containrrr/watchtower#:~:text=We%20do%20not%20recommend%20using%20Watchtower%20in%20a%20commercial%20or%20production%20environment.)
if this is the official advice from the [watchtower](https://github.com/containrrr/watchtower) developers, then we can use it only on Demo (not production).

Reply:
See Slack